views/help/partials/help_about.hbs

```diff
@@ -12,7 +12,7 @@
 				"src": "/help/confluence/23232533"
 			},
 			{
-				"title": "Ansprechpartner",
+				"title": "{{$t "help.partials.heading.contactPerson" }}",
```

Comment:
You can't call a helper inside of a string... 
I would suggest handling this with the helper (concat 'string' ($t "some.key") 'someMorestring')
That would not really be fancy, but it would do the trick.

Reply:
like this?
"title":  `''.concat($t "help.partials.heading.contactPerson")`

Reply:
done ;)